tests/behavioural/features/steps/draft_get.py

```diff
@@ -39,7 +39,7 @@ def update_encrypted_jwt():
     database.db.create_all()
```

Comment:
I think `def update_encrypted_jwt():` is the same in draft_get and draft_post, maybe worth looking at?